TypeCobol.Test/Parser/CodeElements/DISPLAY.rdz.cbl

```diff
@@ -20,3 +20,27 @@
 000000 display toto(1:5) toto toto of mycopy
 000000 display high-value
 000000 display spaces
+000000 display all 'a'
+000000 display all 'a' upon SYSIN with no advancing
```

Comment:
SYSIN is for input , this line won't compile on IBM compiler. Since we don't have a check for that yet, I suggest to turn it into `upon SYSOUT`.

Reply:
Done.
As a consequence there is a one position shift in the corresponding test results.